worker/src/main/java/org/apache/celeborn/service/deploy/worker/memory/MemoryManager.java

```diff
@@ -253,27 +258,24 @@ private MemoryManager(CelebornConf conf) {
 
   public ServingState currentServingState() {
     long memoryUsage = getMemoryUsage();
-    boolean pausePushData = memoryUsage > pausePushDataThreshold;
-    boolean pauseReplicate = memoryUsage > pauseReplicateThreshold;
-    boolean resume = memoryUsage < resumeThreshold;
-    if (pausePushData || pauseReplicate) {
-      underPressure = true;
-    } else if (resume) {
-      underPressure = false;
-    }
-    if (pausePushData && pauseReplicate) {
+    // trigger pause both push and replicate
+    if (memoryUsage > pauseReplicateThreshold) {
```

Comment:
Let's assume that a newbie comes and try to understand the method. Why should this method return PUSH_AND_REPLICATE_PAUSED when memory usage is above pauseReplicateThreshold? Will renaming pauseReplicateThreshold to pausePushAndReplicateThreshold be a better name for understanding?

Reply:
Have same idea, maybe we should can this conf `celeborn.worker.directMemoryRatioToPauseReplicate` => `celeborn.worker.directMemoryRatioToPauseAll`?

WDYT?

Reply:
IMO we can add comment describing the logic so that we don't need to change the config name.

Reply:
added.